packages/meditrak-app/app/assessment/specificQuestions/CalculatedQuestion.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { StyleSheet, View } from 'react-native';
+import { Text } from '../../widgets';
+import { getLineHeight, THEME_FONT_SIZE_ONE } from '../../globalStyles';
+import { getCalculatedResult } from '../selectors';
+
+export class CalculatedQuestionComponent extends PureComponent {
+  componentDidMount() {
+    this.updateAnswer(this.props);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    this.updateAnswer(nextProps);
+  }
+
+  updateAnswer(props) {
+    const { answer, onChangeAnswer, calculatedResult } = props;
+    if (answer !== calculatedResult) {
+      onChangeAnswer(calculatedResult);
+    }
+  }
+
+  render() {
+    const { translatedText, answer } = this.props;
```

Comment:
```suggestion
    const { answerDisplayValue, answer } = this.props;
```
Particularly within the presentational component, I think we can be more specific with the name of this variable